examples/tcp-echo/server.rs

```diff
@@ -34,62 +34,51 @@ pub const SOCK_STREAM: i32 = libc::SOCK_STREAM;
 // Structures
 //======================================================================================================================
 
-/// A TCP echo server.
 pub struct TcpEchoServer {
-    /// Underlying libOS.
     libos: LibOS,
-    /// Local socket descriptor.
-    sockqd: QDesc,
-    /// Set of connected clients.
-    clients: HashSet<QDesc>,
-    /// List of pending operations.
-    qts: Vec<QToken>,
-    /// Reverse lookup table of pending operations.
-    qts_reverse: HashMap<QToken, QDesc>,
+    local_sockqd: QDesc,
```

Comment:
Not super important but maybe this should be better named `listening_qd`

Reply:
Done